### PR TITLE
Creation of CustomerPayment Model

### PR DESCRIPTION
**Pull request by Shu52**
# Description
 Created CustomerPayment Model for intersection table between Customer #1  and PaymentTypes  #3 
 Fixes # (#1 , #3 )
 ## Type of change
 Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
 # Testing Instructions
 -Ensure that Customer.cs and PaymentType.cs is included in the Model folder

-In the directory BangazonAPI, use command "dotnet run" to run program. The server should then start and run without errors.

-Make sure all names line up with ERD.(https://drive.google.com/file/d/1Qo9O2oUezNd13Wsr0_VUbUtwwDo4-oAt/view)
 # Checklist:
 - [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Shu52** [comment]
Let me know if the whitespace is excessive and any other critiques.

File: Models/CustomerPayment.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        public int CardNumber { get; set; }
+
+        [StringLength(3,ErrorMessage = "CCV code is three numbers"),Required]
+        public string CcvCode { get; set; }
+
+        [StringLength(10, MinimumLength = 4,ErrorMessage = "Expiration Date is four to ten numbers"),Required]
+        public string ExpirationDate { get; set;}
+        //Foreign Keys
+        [Required]
+        public int CustomerId { get; set; }
```
Comment (jordan-castelloe) [inline Models/CustomerPayment.cs]: I can't find anything in the tickets that says we'll need to show the the actual `Customer` when we use this model. 

If you include something that's not in the ERD, please reference the ticket number so we can cross-reference.

Comment (jordan-castelloe) [reply Models/CustomerPayment.cs]: (this was in reference to line 22 but I clicked the wrong line number, sorry!)

Comment (Shu52) [reply Models/CustomerPayment.cs]: removed lines as directed

**Pull request merged**